contracts/src/peer_protocol.cairo

```diff
@@ -24,12 +24,22 @@ struct Transaction {
     tx_hash: felt252,
 }
 
+#[derive(Drop, Serde, Copy, starknet::Store)]
```

Comment:
![Uploading Screenshot 2024-11-25 at 21.56.29.png…]()

Reply:
resolving that now